### PR TITLE
CNX-9206 - Adds Navis2025 support

### PR DESCRIPTION
**Pull request by jsdbroughton**
Needs Innosetup files to be updated to match

### CODE REVIEW

**Review by BovineOx — DISMISSED** [review]
I think there's just a debug line that needs to come out...

File: ConnectorNavisworks/ConnectorNavisworks/Entry/Ribbon.xaml.cs [hunk 1]
```diff
@@ -205,6 +205,11 @@ public override int ExecuteCommand(string commandId, params string[] parameters)
 #if NAVMAN21
     buildVersion = "2024";
 #endif
+#if NAVMAN22
+    buildVersion = "2025";
+#endif
+
+    MessageBox.Show(NavisworksApp.Version.RuntimeProductName);
```
[the remark this answers is not in the document]
Comment (jsdbroughton) [reply ConnectorNavisworks/ConnectorNavisworks/Entry/Ribbon.xaml.cs:212]: It's a build variable.

**Review by AlanRynne — APPROVED** [review]
In my eyes... all is good.

File: ConnectorNavisworks/ConnectorNavisworks/Entry/Ribbon.xaml.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsdbroughton) [reply ConnectorNavisworks/ConnectorNavisworks/Entry/Ribbon.xaml.cs]: Ahhwit i see which line you are referring to now - doh

**Review by BovineOx — APPROVED** [review]
Yeah I see how that messagebox was used, I don't think I saw the whole context....

**Pull request merged**